examples/tiny_yolov2/README.md

```diff
@@ -15,7 +15,7 @@ $ go run main.go -h
   -model string
         path to the model file (default "model.onnx")
   -output string
-        path of an output png file (use - for stdout) (default "output.png")
+        path of an output png file (use - for stdout)
```

Comment:
PNG or JPEG ?

Reply:
This is obsolete, as there is no output anymore.

Bu, when an output is specified it must be in png.